libtrue/Makefile

```diff
@@ -3,6 +3,8 @@
 LIB=		true
 SHLIB_MAJOR=	0
 
+LDADD+=		-lxo
```

Comment:
This will fail to compile with my overlink testing.  I plan to commit it soon.

```
(overlinktest) [libtrue.so.0] ==> Checking for unneeded LDADD...
checking if libc.so.7 is needed: yes... __stack_chk_fail
(overlinktest) [libtrue.so.0] ==> FAIL. Unneeded LDADD: xo
*** [.overlinktest] Error code 65
```